examples/source/user-consent/Geolocation-based_Consent_Flow_opt_out.html

```diff
@@ -0,0 +1,170 @@
+<!---
```

Comment:
I wouldn't call this example an opt-out. It's really client-side managed vs server-side managed.

Reply:
Agreed